ui/DesignSystem/Component/Header/Large.svelte

```diff
@@ -117,27 +112,27 @@
       {/if}
 
       <div class="metadata">
-        <div class="user">
-          <h1 data-cy="entity-name" style="display: flex; align-items: center;">
-            {name}
-          </h1>
-        </div>
+        <h1 data-cy="entity-name" class="typo-overflow-ellipsis" title={name}>
+          {name}
+        </h1>
         <ShareableIdentifier {urn} style="margin-top: 0.5rem;" />
         {#if description.length > 0}
-          <p class="description">{description}</p>
+          <p class="description typo-overflow-ellipsis" title={description}>
+            {description}
+          </p>
         {/if}
         {#if stats}
           <div class="project-stats" data-cy="project-stats">
             <div class="project-stat-item">
               <Icon.Branch />
-              <p style="margin-left: 0.5rem;">
+              <p style="margin-left: 0.5rem; white-space: nowrap;">
```

Comment:
Can we move the styles for `p` into `<style>`?

Reply:
https://github.com/radicle-dev/radicle-upstream/pull/1106/commits/2e74eea5fe4d687219cf9314f50d50ef622be4a0